reanalysis/comparison.py

```diff
@@ -58,25 +59,37 @@ def __init__(
         self, chrom: str, pos: int, ref: str, alt: str, confidence: List[Confidence]
     ):
         """
-
+        always normalise contig - upper case, with no CHR prefix
         :param chrom:
         :param pos:
         :param ref:
         :param alt:
         :param confidence:
         """
-        self.chr: str = chrom
+        self.chr: str = chrom.upper().strip('CHR')
```

Comment:
nit: strip('CHR') is not removing the string 'CHR', but is removing any leading or trailing instances of those characters. In this case I can not imagine it ever would do the wrong thing, but I frequently overlook this distinction and it can turn into a foot gun.

Reply:
Aye, I've been stung by that before... without moving to regex `string[string.startswith(prefix) and len(prefix):]` can do it. I'll note this change when I get to this PR